app/javascript/js/controllers/item_select_all_controller.js

```diff
@@ -51,6 +55,36 @@ export default class extends Controller {
     this.selectedAllValue = !this.selectedAllValue
     this.unselectedMessageTarget.classList.toggle('hidden')
     this.selectedMessageTarget.classList.toggle('hidden')
+
+    const resourceIds = JSON.parse(this.element.dataset.selectedResources).join(',')
```

Comment:
This line is duplicated, can't we pass the `this.element` to the `updateLinks` and compute the parse logic inside the function?